helix-term/src/args.rs

```diff
@@ -17,6 +17,7 @@ pub struct Args {
     pub log_file: Option<PathBuf>,
     pub config_file: Option<PathBuf>,
     pub files: Vec<(PathBuf, Position)>,
+    pub line_number: usize,
```

Comment:
It looks like this isn't used outside of the `parse_args` function so it could be a local in that function: `let mut line_number = 0;`